src/components/OnThisPageNav.vue

```diff
@@ -54,6 +62,13 @@ export default {
       url: buildUrl(`#${item.anchor}`, $route.query),
     })),
     currentPageAnchor: ({ store }) => store.state.currentPageAnchor,
+    wordWrapComponent({ role }) {
+      // Articles and Collection pages normally dont need special wrapping for their titles.
+      if (role === TopicRole.collection || role === TopicRole.article) {
```

Comment:
Would it be better to invert this maybe and only use the `WordBreak` for symbol-like roles? I expect that the more common case would be article-like pages, but I may be wrong.

You might have already looked into it and found that this approach works better though—just curious.

Reply:
I was not sure how many other cases we may have, hence why I did it like this.

If you think the `symbol` role is the only one we may need the break for, then I will refactor it, no prob.

Reply:
I looked into this more extensively today, and I think that approach should generally be fine.

The only troubling case would be when a symbol name is used in an overview subheading, but we'll never have enough context to know that an actual symbol is being referenced in that situation, but it should be fairly uncommon and not an awful experience to not have the improved breaks in that situation.

I went ahead and opened #701 for that approach.